api/testdata/react-16.13.0.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "react",
+  "version": "16.13.0",
+  "dependencies": {
+    "loose-envify": {
+      "name": "loose-envify",
+      "version": "1.4.0",
+      "dependencies": {
+        "js-tokens": {
+          "name": "js-tokens",
+          "version": "4.0.0",
+          "dependencies": {}
+        }
+      }
+    },
+    "object-assign": {
+      "name": "object-assign",
+      "version": "4.1.1",
+      "dependencies": {}
+    },
+    "prop-types": {
+      "name": "prop-types",
+      "version": "15.7.2",
```

Comment:
[review] test failing here, we should also add a pipeline that tests on pr commits/merges to master